community/stats/index.md

```diff
@@ -41,14 +41,14 @@ Spinnaker's telemetry project has been reviewed and approved as compliant with t
 
 The above payload is sent to `https://stats.spinnaker.io/log` upon the completion of any pipeline or ad-hoc UI operation.
 
-For release 1.18.x, telemetry is opt-in. Spinnaker administrators can turn **on** data collection by executing
+For release 1.18.x, telemetry is opt-in, as we test and scale the system and fine-tune the dashboards. Spinnaker administrators can help by turning **on** data collection by executing
```

Comment:
I suppose we could use language here similar to below: "...will be disabled by default, "